src/formats/anvil/anvil_world.py

```diff
@@ -134,18 +135,17 @@ def load_region(self, rx: int, rz: int) -> bool:
 
 
 class AnvilWorld(WorldFormat):
-    def __init__(self, directory: str, definitions: str):
+    def __init__(self, directory: str, definitions: str, adapters):
+        super(AnvilWorld, self).__init__(adapters)
         self._directory = directory
         self._materials = DefinitionManager(definitions)
         self._region_manager = _AnvilRegionManager(directory)
-        self.mapping_handler = numpy.array(
-            list(self._materials.blocks.keys()), dtype="object"
-        )
+        self.mapping_handler = BlockManager()
```

Comment:
That should be done in the parent class WorldFormat since every subclass will use that same line.

Reply:
I agree, it does make more to move it there sense it's now more universal for each implementation, I might move line 140-141 to the parent class as well

Reply:
Yeah, that works